[CI] MixedClusterEsqlSpecIT test {date.EvalDateDiffMonthAsWhole1Month ASYNC} failing

**Build Scans:**
- [elasticsearch-periodic #4375 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/jzjmzscy4mona)
- [elasticsearch-periodic #4367 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/htfm4zbfgydlm)
- [elasticsearch-periodic #4363 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/d7ggxm5dis7gy)
- [elasticsearch-periodic #4351 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/qg4ipblihm4ks)
- [elasticsearch-periodic #4347 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/ab6w64s7375km)
- [elasticsearch-periodic #4335 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/irnt66b6efzwc)
- [elasticsearch-periodic #4327 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/tg3fpup337b5k)
- [elasticsearch-periodic #4319 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/vocfjaoqx5ppk)
- [elasticsearch-periodic #4315 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/56t2xyvn6wx7w)
- [elasticsearch-periodic #4307 / 8.14.3_bwc](https://gradle-enterprise.elastic.co/s/wdshcuz46nj5e)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:mixed-cluster:v8.14.3#javaRestTest" -Dtests.class="org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT" -Dtests.method="test {date.EvalDateDiffMonthAsWhole1Month ASYNC}" -Dtests.seed=808C470B71C67B5D -Dtests.bwc=true -Dtests.locale=ig-Latn-NG -Dtests.timezone=Asia/Yekaterinburg -Druntime.java=22
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7Bdate.EvalDateDiffMonthAsWhole1Month%20ASYNC%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
org.elasticsearch.client.ResponseException: method [GET], host [http://[::1]:41481], URI [/_query/async/FjlWQ1FCTjFmVHFDNE5BQWtzVDdTaXccRWlxTmlFdGpUMGVGQVRUOWRHTmFqQToxNDU0MA==?wait_for_completion_timeout=60s&pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
{
  "error" : {
    "root_cause" : [
      {
        "type" : "parsing_exception",
        "reason" : "line -1:0: parsing_exception: line 1:36: token recognition error at: ':'",
        "stack_trace" : "org.elasticsearch.xpack.esql.parser.ParsingException: line -1:0: parsing_exception: line 1:36: token recognition error at: ':'\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$1.syntaxError(EsqlParser.java:109)\n\tat org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:41)\n\tat org.antlr.v4.runtime.Lexer.notifyListeners(Lexer.java:364)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:144)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$ParametrizedTokenSource.nextToken(EsqlParser.java:135)\n\tat org.antlr.v4.runtime.BufferedTokenStream.fetch(BufferedTokenStream.java:169)\n\tat org.antlr.v4.runtime.BufferedTokenStream.sync(BufferedTokenStream.java:152)\n\tat org.antlr.v4.runtime.BufferedTokenStream.consume(BufferedTokenStream.java:136)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.execATN(ParserATNSimulator.java:515)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.adaptivePredict(ParserATNSimulator.java:371)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.booleanExpression(EsqlBaseParser.java:838)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.field(EsqlBaseParser.java:1857)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.fields(EsqlBaseParser.java:1774)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.rowCommand(EsqlBaseParser.java:1720)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.sourceCommand(EsqlBaseParser.java:401)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.query(EsqlBaseParser.java:299)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.singleStatement(EsqlBaseParser.java:201)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.invokeParser(EsqlParser.java:73)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.createStatement(EsqlParser.java:45)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.parse(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:125)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:71)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.innerExecute(TransportEsqlQueryAction.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$execute$1(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:356)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:52)\n\tat org.elasticsearch.xpack.ql.async.AsyncTaskManagementService.asyncExecute(AsyncTaskManagementService.java:188)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:129)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:121)\n\tat org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:100)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:984)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1570)\n\tSuppressed: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: parsing_exception: line 1:36: token recognition error at: ':'\n\t\t... 38 more\n\tCaused by: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: lexer_no_viable_alt_exception: null\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.failOrAccept(LexerATNSimulator.java:309)\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.execATN(LexerATNSimulator.java:230)\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.match(LexerATNSimulator.java:114)\n\t\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:141)\n\t\t... 34 more\n",
        "suppressed" : [
          {
            "type" : "parsing_exception",
            "reason" : "parsing_exception: line 1:36: token recognition error at: ':'",
            "stack_trace" : "org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: parsing_exception: line 1:36: token recognition error at: ':'\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$1.syntaxError(EsqlParser.java:109)\n\tat org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:41)\n\tat org.antlr.v4.runtime.Lexer.notifyListeners(Lexer.java:364)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:144)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$ParametrizedTokenSource.nextToken(EsqlParser.java:135)\n\tat org.antlr.v4.runtime.BufferedTokenStream.fetch(BufferedTokenStream.java:169)\n\tat org.antlr.v4.runtime.BufferedTokenStream.sync(BufferedTokenStream.java:152)\n\tat org.antlr.v4.runtime.BufferedTokenStream.consume(BufferedTokenStream.java:136)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.execATN(ParserATNSimulator.java:515)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.adaptivePredict(ParserATNSimulator.java:371)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.booleanExpression(EsqlBaseParser.java:838)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.field(EsqlBaseParser.java:1857)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.fields(EsqlBaseParser.java:1774)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.rowCommand(EsqlBaseParser.java:1720)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.sourceCommand(EsqlBaseParser.java:401)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.query(EsqlBaseParser.java:299)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.singleStatement(EsqlBaseParser.java:201)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.invokeParser(EsqlParser.java:73)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.createStatement(EsqlParser.java:45)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.parse(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:125)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:71)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.innerExecute(TransportEsqlQueryAction.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$execute$1(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:356)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:52)\n\tat org.elasticsearch.xpack.ql.async.AsyncTaskManagementService.asyncExecute(AsyncTaskManagementService.java:188)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:129)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:121)\n\tat org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:100)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:984)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1570)\nCaused by: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: lexer_no_viable_alt_exception: null\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.failOrAccept(LexerATNSimulator.java:309)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.execATN(LexerATNSimulator.java:230)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.match(LexerATNSimulator.java:114)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:141)\n\t... 34 more\n"
          }
        ]
      }
    ],
    "type" : "parsing_exception",
    "reason" : "line -1:0: parsing_exception: line 1:36: token recognition error at: ':'",
    "stack_trace" : "org.elasticsearch.xpack.esql.parser.ParsingException: line -1:0: parsing_exception: line 1:36: token recognition error at: ':'\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$1.syntaxError(EsqlParser.java:109)\n\tat org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:41)\n\tat org.antlr.v4.runtime.Lexer.notifyListeners(Lexer.java:364)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:144)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$ParametrizedTokenSource.nextToken(EsqlParser.java:135)\n\tat org.antlr.v4.runtime.BufferedTokenStream.fetch(BufferedTokenStream.java:169)\n\tat org.antlr.v4.runtime.BufferedTokenStream.sync(BufferedTokenStream.java:152)\n\tat org.antlr.v4.runtime.BufferedTokenStream.consume(BufferedTokenStream.java:136)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.execATN(ParserATNSimulator.java:515)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.adaptivePredict(ParserATNSimulator.java:371)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.booleanExpression(EsqlBaseParser.java:838)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.field(EsqlBaseParser.java:1857)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.fields(EsqlBaseParser.java:1774)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.rowCommand(EsqlBaseParser.java:1720)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.sourceCommand(EsqlBaseParser.java:401)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.query(EsqlBaseParser.java:299)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.singleStatement(EsqlBaseParser.java:201)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.invokeParser(EsqlParser.java:73)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.createStatement(EsqlParser.java:45)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.parse(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:125)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:71)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.innerExecute(TransportEsqlQueryAction.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$execute$1(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:356)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:52)\n\tat org.elasticsearch.xpack.ql.async.AsyncTaskManagementService.asyncExecute(AsyncTaskManagementService.java:188)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:129)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:121)\n\tat org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:100)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:984)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1570)\n\tSuppressed: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: parsing_exception: line 1:36: token recognition error at: ':'\n\t\t... 38 more\n\tCaused by: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: lexer_no_viable_alt_exception: null\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.failOrAccept(LexerATNSimulator.java:309)\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.execATN(LexerATNSimulator.java:230)\n\t\tat org.antlr.v4.runtime.atn.LexerATNSimulator.match(LexerATNSimulator.java:114)\n\t\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:141)\n\t\t... 34 more\n",
    "suppressed" : [
      {
        "type" : "parsing_exception",
        "reason" : "parsing_exception: line 1:36: token recognition error at: ':'",
        "caused_by" : {
          "type" : "lexer_no_viable_alt_exception",
          "reason" : "lexer_no_viable_alt_exception: null",
          "stack_trace" : "org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: lexer_no_viable_alt_exception: null\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.failOrAccept(LexerATNSimulator.java:309)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.execATN(LexerATNSimulator.java:230)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.match(LexerATNSimulator.java:114)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:141)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$ParametrizedTokenSource.nextToken(EsqlParser.java:135)\n\tat org.antlr.v4.runtime.BufferedTokenStream.fetch(BufferedTokenStream.java:169)\n\tat org.antlr.v4.runtime.BufferedTokenStream.sync(BufferedTokenStream.java:152)\n\tat org.antlr.v4.runtime.BufferedTokenStream.consume(BufferedTokenStream.java:136)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.execATN(ParserATNSimulator.java:515)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.adaptivePredict(ParserATNSimulator.java:371)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.booleanExpression(EsqlBaseParser.java:838)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.field(EsqlBaseParser.java:1857)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.fields(EsqlBaseParser.java:1774)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.rowCommand(EsqlBaseParser.java:1720)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.sourceCommand(EsqlBaseParser.java:401)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.query(EsqlBaseParser.java:299)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.singleStatement(EsqlBaseParser.java:201)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.invokeParser(EsqlParser.java:73)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.createStatement(EsqlParser.java:45)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.parse(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:125)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:71)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.innerExecute(TransportEsqlQueryAction.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$execute$1(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:356)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:52)\n\tat org.elasticsearch.xpack.ql.async.AsyncTaskManagementService.asyncExecute(AsyncTaskManagementService.java:188)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:129)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:121)\n\tat org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:100)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:984)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1570)\n"
        },
        "stack_trace" : "org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: parsing_exception: line 1:36: token recognition error at: ':'\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$1.syntaxError(EsqlParser.java:109)\n\tat org.antlr.v4.runtime.ProxyErrorListener.syntaxError(ProxyErrorListener.java:41)\n\tat org.antlr.v4.runtime.Lexer.notifyListeners(Lexer.java:364)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:144)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser$ParametrizedTokenSource.nextToken(EsqlParser.java:135)\n\tat org.antlr.v4.runtime.BufferedTokenStream.fetch(BufferedTokenStream.java:169)\n\tat org.antlr.v4.runtime.BufferedTokenStream.sync(BufferedTokenStream.java:152)\n\tat org.antlr.v4.runtime.BufferedTokenStream.consume(BufferedTokenStream.java:136)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.execATN(ParserATNSimulator.java:515)\n\tat org.antlr.v4.runtime.atn.ParserATNSimulator.adaptivePredict(ParserATNSimulator.java:371)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.booleanExpression(EsqlBaseParser.java:838)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.field(EsqlBaseParser.java:1857)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.fields(EsqlBaseParser.java:1774)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.rowCommand(EsqlBaseParser.java:1720)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.sourceCommand(EsqlBaseParser.java:401)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.query(EsqlBaseParser.java:299)\n\tat org.elasticsearch.xpack.esql.parser.EsqlBaseParser.singleStatement(EsqlBaseParser.java:201)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.invokeParser(EsqlParser.java:73)\n\tat org.elasticsearch.xpack.esql.parser.EsqlParser.createStatement(EsqlParser.java:45)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.parse(EsqlSession.java:143)\n\tat org.elasticsearch.xpack.esql.session.EsqlSession.execute(EsqlSession.java:125)\n\tat org.elasticsearch.xpack.esql.execution.PlanExecutor.esql(PlanExecutor.java:71)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.innerExecute(TransportEsqlQueryAction.java:160)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$execute$1(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.action.ActionListener.run(ActionListener.java:356)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:143)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.execute(TransportEsqlQueryAction.java:52)\n\tat org.elasticsearch.xpack.ql.async.AsyncTaskManagementService.asyncExecute(AsyncTaskManagementService.java:188)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.doExecuteForked(TransportEsqlQueryAction.java:129)\n\tat org.elasticsearch.xpack.esql.plugin.TransportEsqlQueryAction.lambda$doExecute$0(TransportEsqlQueryAction.java:121)\n\tat org.elasticsearch.action.ActionRunnable$4.doRun(ActionRunnable.java:100)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)\n\tat org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:984)\n\tat org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\n\tat java.lang.Thread.run(Thread.java:1570)\nCaused by: org.elasticsearch.common.io.stream.NotSerializableExceptionWrapper: lexer_no_viable_alt_exception: null\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.failOrAccept(LexerATNSimulator.java:309)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.execATN(LexerATNSimulator.java:230)\n\tat org.antlr.v4.runtime.atn.LexerATNSimulator.match(LexerATNSimulator.java:114)\n\tat org.antlr.v4.runtime.Lexer.nextToken(Lexer.java:141)\n\t... 34 more\n"
      }
    ]
  },
  "status" : 400
}

```

**Issue Reasons:**
- [8.x] 11 failures in test test {date.EvalDateDiffMonthAsWhole1Month ASYNC} (1.4% fail rate in 802 executions)
- [8.x] 11 failures in step 8.14.3_bwc (55.0% fail rate in 20 executions)
- [8.x] 11 failures in pipeline elasticsearch-periodic (55.0% fail rate in 20 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec
@@ -369,7 +369,7 @@ date1:date               | dd_ms:integer
 
 evalDateDiffMonthAsWhole0Months
 
-ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-01-01T00:00:00"::DATETIME
+ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-01-01T00:00:00")
 | EVAL msecs=DATE_DIFF("milliseconds", from, to), months=DATE_DIFF("month", from, to)
 ;
 
@@ -380,7 +380,7 @@ ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-01-01T00:00:00"::DATETIM
 
 evalDateDiffMonthAsWhole1Month
 
-ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-02-01T00:00:00"::DATETIME
+ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-02-01T00:00:00")
 | EVAL secs=DATE_DIFF("seconds", from, to), months=DATE_DIFF("month", from, to)
 ;
 
@@ -392,7 +392,7 @@ ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-02-01T00:00:00"::DATETIM
 evalDateDiffYearAsWhole0Years
 required_capability: date_diff_year_calendarial
 
-ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-01-01T00:00:00"::DATETIME
+ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-01-01T00:00:00")
 | EVAL msecs=DATE_DIFF("milliseconds", from, to), years=DATE_DIFF("year", from, to)
 ;
 
@@ -403,7 +403,7 @@ ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2024-01-01T00:00:00"::DATETIM
 evalDateDiffYearAsWhole1Year
 required_capability: date_diff_year_calendarial
 
-ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2025-01-01T00:00:00"::DATETIME
+ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2025-01-01T00:00:00")
 | EVAL secs=DATE_DIFF("seconds", from, to), years=DATE_DIFF("year", from, to)
 ;
 
@@ -414,7 +414,7 @@ ROW from="2023-12-31T23:59:59.999Z"::DATETIME, to="2025-01-01T00:00:00"::DATETIM
 evalDateDiffYearAsWhole1Year
 required_capability: date_diff_year_calendarial
 
-ROW from="2024-01-01T00:00:00Z"::DATETIME, to="2025-01-01T00:00:00"::DATETIME
+ROW from=TO_DATETIME("2024-01-01T00:00:00Z"), to=TO_DATETIME("2025-01-01T00:00:00")
 | EVAL secs=DATE_DIFF("seconds", from, to), years=DATE_DIFF("year", from, to)
 ;
 
@@ -426,9 +426,9 @@ evalDateDiffYearForDocs
 required_capability: date_diff_year_calendarial
 
 // tag::evalDateDiffYearForDocs[]
-ROW end_23="2023-12-31T23:59:59.999Z"::DATETIME,
-  start_24="2024-01-01T00:00:00.000Z"::DATETIME,
-    end_24="2024-12-31T23:59:59.999"::DATETIME
+ROW end_23=TO_DATETIME("2023-12-31T23:59:59.999Z"),
+  start_24=TO_DATETIME("2024-01-01T00:00:00.000Z"),
+    end_24=TO_DATETIME("2024-12-31T23:59:59.999")
 | EVAL end23_to_start24=DATE_DIFF("year", end_23, start_24)
 | EVAL end23_to_end24=DATE_DIFF("year", end_23, end_24)
 | EVAL start_to_end_24=DATE_DIFF("year", start_24, end_24)

